tests/integration/test_async_http_client.py

```diff
@@ -500,6 +501,20 @@ async def test_get_supply(test_http_client_async):
     assert_valid_response(resp)
 
 
+@pytest.mark.integration
+async def test_get_token_largest_accounts(test_http_client):
+    """Test get token largest accounts."""
+    resp = test_http_client.get_token_largest_accounts(WRAPPED_SOL_MINT)
```

Comment:
Should be `resp = await test_http_client_async.get_token_largest_accounts(WRAPPED_SOL_MINT)`